recipes/openlibm/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "openlibm" %}
+{% set version = "0.5.4" %}
+{% set sha256 = "9a8ae1d17825a4a6a4c013d36a7f4348b27c47eedb6549c521ecc9c79d021c13" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/JuliaLang/openlibm/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - toolchain
+    - pkg-config
+
+test:
+  commands:
+    - test -d "${PREFIX}/include/openlibm"                            # [unix]
+    - test -f "${PREFIX}/lib/libopenlibm.a"                           # [unix]
+    - test -f "${PREFIX}/lib/libopenlibm${SHLIB_EXT}"                 # [unix]
+
+about:
+  home: http://openlibm.org/
+  license: MIT, ISC, BSD 2-Clause
+  license_file: LICENSE.md
+  summary: "High quality system independent, portable, open source libm implementation"
+  dev_url: https://github.com/JuliaLang/openlibm
+
+extra:
+  recipe-maintainers:
+    - dfornika
+    - jakirkham
```

Comment:
Have added you to the maintainers, @dfornika. Please let me know if this is ok.

Reply:
Yes, that's ok.